helpers/types.ts

```diff
@@ -59,6 +59,7 @@ export enum ePolygonNetwork {
 export enum eBobNetwork {
   main = "bob",
   testnet = "bob-sepolia",
+  testnet_new = "bob-sepolia-new",
```

Comment:
Maybe I would do the opposit so `testnet_new` is `testnet` and `bob-sepolia` while we rename the old one to `testnet_old` or `testnet_deprecated` or so

Reply:
totally! Done